run/action.yaml

```diff
@@ -58,9 +58,9 @@ runs:
         DISC_PATH=${{ runner.temp }}/discovery
         DISC_ARC_PATH=${{ runner.temp }}/discovery.tar.zstd
         SSH_AUTH_SOCK=${{ runner.temp }}/ssh_agent.sock
-        NIX_KEY_PATH=${{ inputs.key_path }}
         BUILDER=${{ inputs.builder }}
         CACHE=${{ inputs.cache }}
+        NIX_KEY_PATH=
```

Comment:
This doesn't actually change anything, if the input is empty this is what would happen anyway

Reply:
No the input has a default to the runner.temp, specifically a file that doesn't exist unless nix_key is set

Reply:
I've to say I also wondered where that default would be coming from, though.

Reply:
Ah here it is: https://github.com/divnix/std-action/blob/main/run/action.yaml#L8